code/modules/client/preferences_gear.dm

```diff
@@ -1241,6 +1253,12 @@ var/global/list/gear_datums = list()
 	sort_category = "misc"
 	cost = 1
 
+/datum/gear/cape
```

Comment:
Erm, won't this give everyone a larger-than-average backpack? At the cost of almost nothing?